platform-configs/protectli-v1210.robot

```diff
@@ -10,3 +10,7 @@ ${CPU_MIN_FREQUENCY}=           800
 ${PLATFORM_CPU_SPEED}=          2.00
 ${PLATFORM_RAM_SPEED}=          2933
 ${PLATFORM_RAM_SIZE}=           4096
+
+# List of ethernet interfaces
+&{ETH_PORTS}=                   eno0:=64-62-66-2f-00-12
+...                             eno1:=64-62-66-2f-00-13
```

Comment:
We should only care about MACs. MACs are immutable, but the interface names are and the test aims to detect this. We cannot bind MACs to enoX. We just need the order of MACs to appear in a specific order dictated by the labels on the chassis.
```suggestion
&{ETH_PORTS}=                   64-62-66-2f-00-12
...                             64-62-66-2f-00-13
```

Apply it to the other configs as well and modify test accordingly.

Reply:
makes sense https://github.com/Dasharo/open-source-firmware-validation/pull/515/commits/82760ad9e19d8348e16b3bda7cab450ad73a8418